tests/travis/checkout_test_against_branch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPT_DIR=`dirname $0`
+
+if [ "$TEST_AGAINST_PIWIK_BRANCH" == "" ]; then
+    if [ "$TEST_AGAINST_CORE" == "latest_stable" ]; then # test against the latest stable release of piwik core (including betas & release candidates)
```

Comment:
Maybe we rename `latest_stable` -> `latest_piwik_beta_channel` 
(consistent with how we call it the beta channel)

maybe this feature won't be used at all later?
the idea would be to use `minimum_required_piwik` in all plugins (via the travis.yml auto updater running next sunday?)